lib/hooks/api/useContributors.ts

```diff
@@ -27,7 +27,7 @@ const useContributors = (limit = 10, repoIds: number[] = []) => {
   const filterQuery = getFilterQuery(selectedFilter);
   const query = new URLSearchParams(filterQuery);
 
-  if (Number.isNaN(Number(topic))) {
+  if (Number.isNaN(Number(topic)) && topic !== undefined) {
```

Comment:
Does this mean some topics are words instead of numbers?

Reply:
Yes.

For explore pages, its /{pageId/topic}/dashboard -> pageId = javascript
For insight pages its /pages/{username}/{pageId/topic}/dashboard -> pageId = 42